app/main.js

```diff
@@ -33,12 +33,16 @@ async function main() {
 
     const airport = getAirport(options.icao);
     let playbook = null;
-    try {
-      const fsPlaybook = fs.readFileSync(`${playbooksDir}/${airport.iso_country}.json`, 'utf8');
-      playbook = JSON.parse(fsPlaybook);
-    } catch (error) {
-      logger.error(`Playbook for ${airport.iso_country} not found`, { type: 'general' });
-      logger.error(error);
+    if (airport) {
+      try {
+        const fsPlaybook = fs.readFileSync(`${playbooksDir}/${airport.iso_country}.json`, 'utf8');
+        playbook = JSON.parse(fsPlaybook);
+      } catch (error) {
+        logger.error(`Playbook for ${airport.iso_country} not found`, { type: 'general' });
+        logger.error(error);
+      }
+    } else {
+      logger.error(`No airport with icao code ${options.icao} found`, { type: 'general' });
     }
     if (playbook) {
```

Comment:
Lines 47-49 (`if (playbook)`) should also be inside the `if (airport)` block.

Reply:
If airport is found then playbook will also be found, else playbook will be null.

Reply:
But that makes sense I will fix this